### PR TITLE
WRN-18225: Added base sample app for React18 POC

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [ ] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added base sample app for React18 POC

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-18225

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

File: sandstone/react-18/resources/ilibmanifest.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"files": []
+}
```
Comment (stanca-pop-lgp) [inline sandstone/react-18/resources/ilibmanifest.json:3]: Please leave empty line at the end.

File: sandstone/react-18/src/App/App.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import {Header} from '@enact/sandstone/Panels';
+import TabLayout, {Tab} from '@enact/sandstone/TabLayout';
+import ThemeDecorator from '@enact/sandstone/ThemeDecorator';
+import classnames from 'classnames';
```
Comment (stanca-pop-lgp) [inline sandstone/react-18/src/App/App.js:4]: Please move this at the beginning.

File: sandstone/react-18/src/views/Batching/Batching.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import Button from '@enact/sandstone/Button';
+import kind from '@enact/core/kind';
```
Comment (stanca-pop-lgp) [inline sandstone/react-18/src/views/Batching/Batching.js:2]: Please sort.

File: sandstone/react-18/src/views/Suspense/Suspense.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import Button from '@enact/sandstone/Button';
+import kind from '@enact/core/kind';
```
Comment (stanca-pop-lgp) [inline sandstone/react-18/src/views/Suspense/Suspense.js:2]: Please sort.

File: sandstone/react-18/src/views/UseTransition/UseTransition.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import Button from '@enact/sandstone/Button';
+import kind from '@enact/core/kind';
```
Comment (stanca-pop-lgp) [inline sandstone/react-18/src/views/UseTransition/UseTransition.js:2]: Please sort.

**Review by stanca-pop-lgp — CHANGES_REQUESTED** [review]

File: sandstone/react-18/src/App/App.js (hunk 2 above)
Comment (daniel-stoian-lgp) [reply sandstone/react-18/src/App/App.js:4]: this is already sorted

**Review by stanca-pop-lgp — APPROVED** [review]
Looks great to me 🥇

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Pull request merged**